src/api/l_data_soundData.c

```diff
@@ -9,8 +9,32 @@ static int l_lovrSoundDataGetBlob(lua_State* L) {
   return 1;
 }
 
+static int l_lovrSoundDataAppend(lua_State* L) {
+  SoundData* soundData = luax_checktype(L, 1, SoundData);
+  Blob* blob = luax_totype(L, 2, Blob);
+  SoundData* sound = luax_totype(L, 2, SoundData);
+  lovrAssert(blob || sound, "Invalid blob appended");
+  size_t appendedSamples = false;
```

Comment:
Wait is this a number or a boolean?

Reply:
oops :S it used to be bool now it's number